tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -133,6 +133,22 @@ ROOT::Experimental::Detail::RPageSinkFile::CommitPageImpl(ColumnHandle_t columnH
    return result;
 }
 
+ROOT::Experimental::RClusterDescriptor::RLocator
+ROOT::Experimental::Detail::RPageSinkFile::WriteRawPageImpl(
+   ROOT::Experimental::Detail::RPageStorage::RRawPage page)
+{
+   // todo(max) check if setting packedBytes to zippedBytes breaks blob reads
+   auto offsetData = fWriter->WriteBlob(page.fBuffer.get(),
+      /* zippedBytes */ page.fSize, /* packedBytes */ page.fSize);
```

Comment:
I would like to double-check this behaviour is OK (setting `packedBytes` to a possibly incorrect value). At this point we have no way of finding packedBytes without uncompressing the page. `packedBytes` doesn't seem to be used during page reads but I would like to flag this for special consideration during review.

Reply:
Hm, zipped and unzipped bytes (packed bytes) should be used during reading in order to figure out whether or not the page is compressed. If and only if compression doesn't make the page smaller, than the uncompressed page is stored and `packedBytes == zippedBytes`.